pkgs/applications/editors/rstudio/default.nix

```diff
@@ -94,30 +119,30 @@ stdenv.mkDerivation rec {
       soci
       sqlite.dev
     ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_11 ]
     ++ lib.optionals server [ pam ]
-    ++ lib.optionals (!server) [
-      libsForQt5.qtbase
-      libsForQt5.qtxmlpatterns
-      libsForQt5.qtsensors
-      libsForQt5.qtwebengine
-      libsForQt5.qtwebchannel
-    ];
+    ++ lib.optionals (!server) [ fontconfig ];
 
   cmakeFlags =
     [
-      (lib.cmakeFeature "RSTUDIO_TARGET" (if server then "Server" else "Desktop"))
+      (lib.cmakeFeature "RSTUDIO_TARGET" (if server then "Server" else "Electron"))
       (lib.cmakeBool "RSTUDIO_USE_SYSTEM_SOCI" true)
       (lib.cmakeBool "RSTUDIO_USE_SYSTEM_BOOST" true)
       (lib.cmakeBool "RSTUDIO_USE_SYSTEM_YAML_CPP" true)
       (lib.cmakeBool "RSTUDIO_DISABLE_CHECK_FOR_UPDATES" true)
       (lib.cmakeBool "QUARTO_ENABLED" true)
-      (lib.cmakeFeature "CMAKE_INSTALL_PREFIX" "${placeholder "out"}/lib/rstudio")
+      (lib.cmakeBool "RSTUDIO_CRASHPAD_ENABLED" false) # need to explicitly disable for x86_64-darwin
```

Comment:
This is disabled for all architectures not just x86_64-darwin, should this be a conditional based on platform?

Reply:
This should option should be disabled on all platforms.
The comment is there because the cmake variable is only ON by default on x86_64-darwin.
I'll re-check it once I get to a computer, but that's what I remember.

Reply:
Reworded the comment